### DISCUSSION TITLE
How to delete Pull-Request

### DISCUSSION THREAD

**Discussion by SeniorDeveloper94** [community/community#29383] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, Every one.

I want to delete pull request on an inactive fork repo by an inactive user.
How can I do for this?

Many thanks.

**Comment by K-0817** [comment, marked as answer]
Hi
I think pull requests can't be deleted through the UI.
If you request, GitHub will delete pull requests when they contain sensitive information like passwords or other credentials. You can contact GitHub support here: https://support.github.com/contact.

**Reply by SeniorDeveloper94** [reply]
Thanks for your fast responsive.

**Reply by SeniorDeveloper94** [reply]
I will have contact with github support about this.
Thanks again.

**Comment by K-0817** [comment]
Welcom to you any time.
